bandit/plugins/zipfile_extractall.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding:utf-8 -*-
```

Comment:
This is only necessary if there are unicode characters in this source file, which there are not.